src/main/java/frc/robot/commands/ConveyorCommands/ConveyFireCommand.java

```diff
@@ -1,10 +1,12 @@
 package frc.robot.commands.ConveyorCommands;
 
 import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.subsystems.BlinkinSubsystem;
 import frc.robot.subsystems.ConveyorSubsystem;
 
 public class ConveyFireCommand extends Command{
     ConveyorSubsystem conveyor; 
+    BlinkinSubsystem blinkin;
```

Comment:
This is not being initialized anywhere. You need to pass an instance of `BlinkinSubsystem` into the constructor. This applies to all of the commands in this PR